Source/WebCore/inspector/NetworkResourcesData.cpp

```diff
@@ -256,10 +256,18 @@ void NetworkResourcesData::maybeDecodeDataToContent(const String& requestId)
     if (!resourceData->hasData())
         return;
 
-    m_contentSize += resourceData->decodeDataToContent();
-    size_t dataLength = resourceData->content().sizeInBytes();
-    if (dataLength > m_maximumSingleResourceContentSize)
-        m_contentSize -= resourceData->evictContent();
+    auto dataLength = resourceData->dataLength();
+    m_contentSize -= dataLength;
+
+    resourceData->decodeDataToContent();
+    dataLength = resourceData->content().sizeInBytes();
```

Comment:
NIT: rather than reassign to `dataLength` could we instead have this be called `byteCount` or something?